http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ZombieClusterMonitor.scala

```diff
@@ -101,6 +102,10 @@ class ZombieClusterMonitor(config: ZombieClusterConfig,
     (googleProjectDAO.isProjectActive(googleProject.value), googleProjectDAO.isBillingActive(googleProject.value))
       .mapN(_ && _)
       .recover {
+        //if we fail because of a permission error, we consider the cluster a zombie, as the permissions have been clean-up elsewhere
```

Comment:
can this be described better?

Reply:
Maybe:
```
// Consider the cluster a zombie if the Leo service account no longer has permission to access its project. This can happen when free-credits projects expire.
```